Sources/MapboxMaps/Style/Style.swift

```diff
@@ -150,11 +150,11 @@ public final class Style: StyleProtocol {
     // MARK: - Sources
 
     /**
-     Adds a source to the map
+     Adds a `source` to the map
      - Parameter source: The source to add to the map.
      - Parameter identifier: A unique source identifier.
 
-     - Throws: StyleError or type conversion errors
+     - Throws: `StyleError` or type conversion errors
```

Comment:
```suggestion
     - Throws: ``StyleError`` if there is a problem adding the `source`.
```
iirc this function does not throw `TypeConversionError`